### ISSUE TITLE
Newsletter menu in dashboard needs a better control or label.

### ISSUE THREAD

**Issue by rkingett** [buttondown/roadmap#2340]
The menu that appears at the very top of the page in the DOM which says the newsletter name has certain areas where the keyboard can't activate the menu, especially if the newsletter has a lengthy title. This should be changed to a proper button or other accessible menu item element to make sure it has a larger activation area on desktop and mobile.
